app/src/main/res/layout/activity_login.xml

```diff
@@ -5,89 +5,96 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:fillViewport="true"
+    android:background="@color/colorPrimary"
     tools:context=".login.views.LoginActivity">
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="0dp">
 
-        <LinearLayout
-            android:id="@+id/login_container"
-            android:layout_width="0dp"
+        <ImageView
+            android:id="@+id/login_logo"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginStart="24dp"
+            android:src="@drawable/ic_logo"
+            android:adjustViewBounds="true"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_marginStart="@dimen/default_margin_giant"
+            android:layout_marginEnd="@dimen/default_margin_giant"
+            android:layout_marginTop="@dimen/default_margin_giant"
+            />
+
+        <androidx.cardview.widget.CardView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/default_margin_large"
             android:layout_marginTop="@dimen/default_margin_enormous"
-            android:layout_marginEnd="24dp"
-            android:gravity="center"
-            android:orientation="vertical"
+            android:layout_marginEnd="@dimen/default_margin_large"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent">
-
-            <TextView
-                android:id="@+id/tv_title"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="@dimen/default_margin_enormous"
-                android:text="@string/studs"
-                android:textSize="62sp"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintTop_toBottomOf="@id/login_logo"
+            app:cardCornerRadius="15dp"
+            android:elevation="15dp">
```

Comment:
This is a lot of elevation. https://material.io/design/environment/elevation.html

Should be between 1dp to 8dp.